app/models/package_manager/go.rb

```diff
@@ -254,7 +269,8 @@ def self.project_find_names(name)
         Rails.cache.write("unreachable-go-hosts:#{host}", true, ex: 1.day)
         Bugsnag.notify(e)
         [name]
-      rescue StandardError
+      rescue StandardError => e
+        Bugsnag.notify(e)
```

Comment:
seems weird to swallow these entirely, we should probably be aware of generic exceptions.